resources/logfs/README.md

```diff
@@ -23,3 +23,96 @@ An equivalently named HTML file is created for each `sample-*.txt` file in this
+      the storage data is blank, with no residual left overs from
+      previous data logging runs.
+- **sample-trailer-dirty**: The same as the sample-trailer, but the DAPLink
+  storage was first filled, and then soft-marked as erased. So the old data is
+  still present on the raw blob, but marked as erased.
+    - This HTML should render the same data table as `sample-trailer`.
+- **sample-trailer-noheader**: The same as the sample-trailer, but manually
+  modified to remove the table header.
+    - This HTML should render the same data table as `sample-trailer`, but
+      without the table header (`Time (seconds)`, `X`, `Y`, `Z`).
+    - At the time of writing the header.html styles the first table row with
+      bold text, so the first data row entry will look like a table header.
+    - Technically, using the CODAL API (and therefore MakeCode or MicroPython)
+      shouldn't allow this to happen, as each log entry needs to be paired to a
+      column, so if this test fails in the future we might consider removing it.
+- **sample-single-column-noheader**: A single column table without a header
+  and with 33 data rows.
+    - The MakeCode programme to generate this test data leaves the `column`
+      fields empty, so the table header contains no entries, and the data table
+      in the raw data blob starts with a LF (`\n`).
+    - At the time of writing the header.html styles the first row, which is
+      normally the table header row, with bold text.
+      In this case, as header is empty, the generated HTML starts with an empty
+      `<tr></tr>`, which is not visible in Chrome and therefore it looks like
+      the HTML renders a table without a header.
+- **sample-time-series**: A valid time series table with 4 columns (Time, x,
+  y , z) and 172 data rows with increasing time stamps.
+    - The raw data contains old logging data marked as erased, that should not
+      be rendered in the HTML table.
+- **sample-invalid-time-series.txt**: An invalid time series table with 4 columns (Goat, x,
+  y , z) and 172 data rows with increasing time stamps.
+    - For a time series the first column should be named `Time...`, this test
+      file was based on `sample-time-series` and the `Time (seconds)` column
+      name has been renamed to `Goat (seconds)`.
+    - In this test file the "Visual Preview" should not work.
+- **sample-time-series-nonsequential**: Based on the `sample-time-series` file,
+  a single entry has been modified to make the time stamps non-sequential.
+    - Data entry 169 (6th from the bottom) has been changed from `18.99` to
+      `99.99`. As the next entry time stamp is `19.1`, this represent a second
+      set of logged data after a micro:bit reset.
+    - In this test file the "Visual Preview" should plot a graph until entry
+      169, and display some kind of warning/error message to indicate it is
+      not plotting the entire dataset.
+- **sample-full-log**: A full log with 6064 rows.
```

Comment:
Maybe a comment to say there should be an indicator showing the log is full?

Reply:
Yep, good idea! 👍